general_rules/TeamRegistration.tex

```diff
@@ -108,10 +108,39 @@ \subsection{Qualification}
 	\item the contributions to the RoboCup@Home league.
 \end{itemize}
 
-\paragraph{Important note to Standard Platform Leagues:} Only unmodified robots may compete in Standard Platform Leagues. Any \textit{slight} modification made to the robot found in the Qualification Material will automatically disqualify the team, for which registration to the international competition will not be possible  (See~\refsec{rule:spl-mods}).
-
 % For getting considered in the evaluation, be sure to insert your team's name when adding information to the \iterm{RoboCup\char64Home Wiki}.
 
+\paragraph{Important note to Standard Platform Leagues:} Only unmodified robots may compete in Standard Platform Leagues. Any \textit{slight} modification made to the robot found in the Qualification Material will automatically disqualify the team, for which registration to the international competition will not be possible  (See~\refsec{rule:spl-mods}).
+
+\subsection{Participation confirmation}
+\label{rule:participation-confirmation}
+In order to max out the number of participating teams, qualified teams \emph{must} contact the \iaterm{Organizing Committee}{OC} to confirm (or cancel) participation.
+
+Confirming attendance implies that the team has sufficient resources to register, and commits to attend to the event.
+Qualified teams missing the \iterm{Participation Confirmation} may be disqualified.
+
+\subsection{Task Selection Announcement}
+\label{rule:task-selection-announcement}
+To facilitate scheduling of test slots during the competition, teams are required to notify the \iaterm{Organizing Committee}{OC} which tasks will be tested in each scenario/stage, at least one month before the competition.
+The tasks a team can select should be according to the number and type of test slots of the competition (see~\Cref{rule:schedule,tbl:max-elegible-tests}).
+
+\begin{table}
+\centering
+\caption{Maximum number of elegible tests in Sydney 2019}
+\label{tbl:max-elegible-tests}
+\newcolumntype{C}[1]{>{\centering\arraybackslash}p{#1}}
+\begin{tabularx}{0.6\linewidth}{X C{0.2\linewidth} C{0.2\linewidth}}
+	\toprule
+	                  & Housekeeper & Party Host\\
```

Comment:
As there are 6 slots in stage 1. Would you allow teams to 4 HK and 2 PH (or vice-versa)? Or would you want to limit it to equal numbers? (Stage 2 doesn't have 'free space'.)